src/nvim/normal.c

```diff
@@ -1468,6 +1468,32 @@ static void move_tab_to_mouse(void)
   }
 }
 
+/// Check that mouse support is enabled for visual mode
+static bool mouse_supports_visual(void)
+{
+  if (!*p_mouse) {
```

Comment:
If wanted, I can do this:
```diff
-  if (!*p_mouse) {
+  if (p_mouse[0] == '\0') {
```
I've seen code doing that in neovim.

Reply:
either one is fine.